index.html

```diff
@@ -19,13 +19,21 @@
        <section class="header-section">
     <header class="header">
       <h2 class="logo-text">My Logo</h2>
-      <nav>
-        <ul>
-          <li><a href="">Portoflio</a></li>
-          <li><a href="">About</a></li>
-          <li><a href="">Contact</a></li>
+      <nav class="navbar"> 
+       
+        <ul class="nav-menu">
+            <ul class="nav-list"><a href="#" class="anchor">Portfolio</a></ul>
+            <ul class="nav-list"><a href="#" class="anchor">About</a></ul>
+            <ul class="nav-list"><a href="#" class="anchor">Contact</a></ul>
+
         </ul>
-      </nav>
+        <div class="hamburger-btn">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+
+    </nav>
```

Comment:
- [ ] As stated in the project requirement kindly make sure that When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

Reply:
Hello @AbdifatahYasin1 for example when i click on About menu it must redirect me to "About " section of page.(            <ul class="nav-list"><a href="**#about**" class="anchor">About</a></ul>)by giving the an id for about section .